.github/workflows/ci.yml

```diff
@@ -26,9 +26,6 @@ jobs:
         env:
           MYSQL_DATABASE: warehouse_test
           MYSQL_ROOT_PASSWORD: warehouse
-        command:
-          - --character-set-server=utf8
-          - --collation-server=utf8_general_ci
```

Comment:
We need to figure out how to add this to CI. It's used in our production database and effects how UUIDs are stored / retrieved.